apps/masterbots.ai/app/actions/ai-main-call.ts

```
@@ -149,6 +136,36 @@ export async function processWithAi(
+export async function processWithAiObject(
+  prompt: string,
+  clientType: AiClientType,
+  model: string,
+): Promise<JSONResponseStream> {
+  try {
+    const messages = [{ role: 'user', content: prompt }] as OpenAI.ChatCompletionMessageParam[]
+
+    const response = await createResponseStreamObject(mbObjectSchema.metadata, {
+      model: AIModels.Default,
+      messages,
+    } as any)
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)